File: c++/evenodd.cpp
#include<iostream>
using namespace std;
int main()
{
    int n;
    cout<<"Enter your num";
    cin>>n;
    if(n!=0)
    {
    if(n%2==0)
    {
        cout<<"Even";
    }
    else if(n%2!= 0)
    {
        cout<<"odd";
    }
    }
    else
    {
        cout<<"num is zero";
    }
    }

File: MyFirstProj/src/EvenOdd.java
import java.util.Scanner;
public class EvenOdd {
    public static void main(String[] args) {
        int num;
    Scanner sc= new Scanner(System.in);
    System.out.println("Enter the num to determine whether the number is even or odd");
    num=sc.nextInt();
    if(num%2==0)
    {
        System.out.println("Even");
    }
    if (num%2!=0) 
    {
    System.out.println("odd");    
 }
 sc.close();
}
}